proposals/infra/INF-0004-register-types-and-diagnostics.md

```diff
@@ -30,11 +30,14 @@ invalid register types are out of date or invalid. For example, in the case of:
 
 `float b : register(u4);`
 an error will be emitted recommending the use of the 'b, c, or i' register
-type, however these register types are no longer in support. Additionally,
-it is possible the user is unaware that this resource won't actually be
-used, but the compiler doesn't communicate that to the user.
-It would be great for any HLSL developer to immediately have answers to 
-the following questions:
+type. However the 'b' and 'i' register types are no longer in support. It
+is worth noting that there is an overloading of the register(...) keyword
+that applies to constant buffer variables using the 'c' register type, as

```

Comment:
Not "constant buffer variables", since that's still confusing (`ConstantBuffer<Foo> myCB` looks like a constant buffer variable).  How about:
```suggestion
using the 'c' register type to indicate an offset into a constant buffer, as
```